ui/raidboss/data/triggers/o4n.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+
+// O4n - Deltascape 4.0 Normal
+
+[{
+
+  zoneRegex: /Deltascape \(V4.0\)/,
+  timelineFile: 'o4n.txt',
+  triggers: [
+    {
+      id: 'O4n Doom',
```

Comment:
this is extremely minor, but I think all the other normal modes are `O4N` and not `O4n`.

Reply:
Entirely correct.